mappings/servers/ezpvp/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "ezpvplaughingoutloud",
+    "name": "EzPvP Network",
+    "description": "Kit-PvP and Practice Network based in europe",
+    "addresses": [
+        "ezpvp.lol",
+        "jamals.lol"
+    ],
+    "primaryAddress": "ezpvp.lol",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#AF15CA",
+    "secondaryColor": "#FFFFFF",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU"
+    ],
+    "gameTypes": [
+        "PvP"
+    ],
+    "website": "ezpvp.lol",
+    "socials": {
+        "telegram": "EzPvP Practice"

```

Comment:
We need a Telegram Slug, which should not have spaces. This would be what you'd see if you pasted the URL invitation.